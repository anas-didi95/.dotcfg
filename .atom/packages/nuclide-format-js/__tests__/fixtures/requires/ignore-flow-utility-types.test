const clubs: $Keys<Class<Suit>> = 'Clubs';
