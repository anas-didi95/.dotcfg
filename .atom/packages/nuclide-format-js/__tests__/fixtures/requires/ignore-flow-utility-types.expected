import type Suit from 'Suit';

const clubs: $Keys<Class<Suit>> = 'Clubs';
